src/TemperatureController.cpp

```diff
@@ -71,7 +72,7 @@ void TemperatureController::handleTemperatureControl(const Command& command,
 			break;
 		}
 		default:
-			CLP_LOG(1, F("Unkown command: %02X\r\n"), command.command);
+			CLP_LOG(1, F("Unkown command\r\n"));
```

Comment:
why you removed the command from the output, this makes it hard to debug these commands

Reply:
It became redundant because of line 22. The logs are now delivered in two parts. One before the switch which contains the hex value (and no \r\n) and the one inside the switch which has the description and completes the line.